qcengine/programs/gamess/runner.py

```diff
@@ -107,7 +107,18 @@ def build_input(
         # Handle conversion from schema (flat key/value) keywords into local format
         optcmd = format_keywords(opts)
 
-        gamessrec["infiles"]["gamess.inp"] = optcmd + molcmd
+        # Currently passing in the entire GAMESS-formatted EFP command (i.e. the $EFRAG section)
+        # In the future, should be passed in as actual data fragments, coordinates, etc.)
+        if input_model.molecule.extras is not None and "efp" in input_model.molecule.extras:
+            efpcmd = input_model.molecule.extras["efp"]
+            gamessrec["infiles"]["gamess.inp"] = optcmd + efpcmd + molcmd
+        else:
+            gamessrec["infiles"]["gamess.inp"] = optcmd + molcmd
```

Comment:
what's in `molecule.extras["efp"]` again? I don't recall it from https://github.com/MolSSI/QCElemental/pull/124#issuecomment-527462426 (though that needn't be a strict blueprint).

Reply:
I suppose I should rename that variable from `molecule.extras["efp"]` to `molecule.extras["efp_molecule"]` in order to be more consistent with the blueprint. In the blueprint, `molecule.extras["efp_molecule"]` is a schema-like dictionary defining all of the fragment locations. In this PR, `molecule.extras["efp_molecule"]` is instead expected to be a GAMESS-ready string, which is directly inserted into the input file. 

This is a short-tem solution. Eventually, we need an input-parsing function that takes in your `efp_molecule` and converts it to the GAMESS-ready string. I held off on doing this right away because I wasn't sure how final the blueprint was.

I added an example script to the PR description that demonstrates how you'd run MAKEFP and then get an EFP interaction energy. This will give you a better idea of what the `efp_molecule` string looks like in this PR.